tests/getdos_TestCase2.py

```diff
@@ -16,7 +17,20 @@
 interactive = False
 
 datadir = os.path.join(os.path.dirname(__file__), "data")
-dataurls = imp.load_source("dataurls", os.path.join(datadir, "dataurls.py"))
+
+
+def load_source(modname, filename):
```

Comment:
the function looks the same as above. It can be included in one spot and imported everywhere.

Reply:
These have been fixed. The load_source function is now in multiphonon.sqe.__init__.py